internal/validate/email.go

```diff
@@ -28,6 +29,10 @@ func (e email) Validate() *Failure {
 	if addr.Name != "" {
 		return fail(e.name, fmt.Sprintf("email address must not contain display name %q", addr.Name))
 	}
+	// domain must contain at least one dot. ParseAddress already checks that it's not leading or trailing
+	if !strings.Contains(strings.Split(addr.Address, "@")[1], ".") {
```

Comment:
The example for the mail address shows the name being included as well:
```
// Address represents a single mail address.
// An address such as "Barry Gibbs <bg@example.com>" is represented
// as Address{Name: "Barry Gibbs", Address: "bg@example.com"}.
type Address struct {
	Name    string // Proper name; may be empty.
	Address string // user@domain
}
```

Looks like that would mean `user.name@domain` would pass this validation when we dont want it to.

Reply:
Not sure what you mean, bruce. This check splits on @ first, takes the domain, then looks for `.`

Reply:
Ah, my bad, read this wrong